civiform/civiform
# [Program Migration] Set up basic import UI

### Description

Part of epic https://github.com/civiform/civiform/issues/7087 to enable admins to export and import programs between instances. This PR sets up an "Import" tab on the admin side. The tab displays a text area that lets admins paste in JSON file contents, and the tab will also display the JSON content once it's been uploaded. A future PR will actually parse that JSON into a `ProgramDefinition` object.

## Release notes

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


#### New Features


### Instructions for manual testing

1. Enable the PROGRAM_MIGRATION_ENABLED flag.
2. As an admin, open the "Import" tab.
3. Paste in some JSON.
4. Click "Display program information".
5. Verify the JSON is displayed.

### Issue(s) this completes

Part of epic #7087
